src/guide/get-started/install-iroha-2.md

```diff
@@ -0,0 +1,75 @@
+# Install Iroha 2
+
+This tutorial guides you through the steps to install Iroha 2 and its necessary binaries on your machine.
+
+## 1. Prerequisites
+
+To install Iroha 2, you need to set up the following first:
+- [git](https://githowto.com/)
+- [OpenSSL](https://www.openssl.org/)
+- [Rust Toolchain](https://www.rust-lang.org/tools/install) (v1.60.0 or newer)
+
+::: details TIP: How to Install OpenSSL
+
+Note that in most Linux setups OpenSSL is already available to you.
+
+- Install OpenSSL on Ubuntu:
+
+  ```bash
+  $ sudo apt-get install libssl-dev
+  ```
+
+- Install OpenSSL on macOS using [brew](https://brew.sh/):
+
+  ```bash
+  $ brew install openssl
+  ```
+
+Check the [OpenSSL installation guide](https://github.com/openssl/openssl/blob/master/INSTALL.md) for details.
+
+:::
+
+
+## 2. Clone Iroha from GitHub
+
+1. Create a new directory for Iroha:
+
+   ```bash
+   $ mkdir -p ~/Git
+   ```
+
+   ::: tip
+
+   On macOS, if you get the `fatal: could not create work tree dir 'iroha': Read-only file system` error, that is because the home directory is not a real file system.
+
+   To fix this, create a directory named `Git`.
```

Comment:
`Git` is not a special name, maybe rephrase this to make it clearer? There has been slight confusion over this: https://github.com/hyperledger/iroha-2-docs/pull/479

Reply:
fixed in commit 68ae813491f1d7b0c5bdaa8350b44d17772179f8